tools/reviewer/webapp/src/app/dashboard/diff/diff-discussion/code-threads/code-threads.component.html

```diff
@@ -4,9 +4,10 @@
 <div class="file-groups">
   <!-- List of groups -->
   <div class="file-group" *ngFor="let fileGroup of fileGroupList; let fileGroupIndex = index;">
-    <div class="filename">
-      <a (mouseup)="openFile($event, fileGroup)">{{ getFileLabel(fileGroup) }}</a>
-    </div>
+    <a
+      class="filename"
+      [routerLink]="fileGroup[0].getFile().getFilenameWithRepo()"
```

Comment:
is there no way around accessing the array index like that? If it's necessary can we put a check if `fileGroup` array exists.

Reply:
We have list of threads, which is grouped by filename. And we need to get the filename from the list.
Each thread contain filename inside. So we can take any thread from the list.
And no need to check existence, because the list is auto-generated. It always contain at least one element.